src/SprykerEco/Zed/PunchoutCatalogs/Business/Reader/PunchoutCatalogsReader.php

```diff
@@ -65,4 +66,24 @@ public function findConnectionByIdWithPassword(int $idConnection): ?PunchoutCata
 
         return $punchoutCatalogConnectionTransfer->setPassword($password);
     }
+
+    /**
+     * @param int $fkCompanyBusinessUnit
+     *
+     * @return \Generated\Shared\Transfer\PunchoutCatalogConnectionCollectionTransfer|null
+     */
+    public function findConnectionByFkCompanyBusinessUnit(
```

Comment:
please use `id` in naming
here and everywhere else

Reply:
Changed.